### PR TITLE
[NPU] Update qwen2 compile config

### PR DESCRIPTION
**Pull request by rnwang04**
### 1. Why the change?

https://github.com/analytics-zoo/nano/issues/1719#issuecomment-2469530491

### 2. User API changes

No change.

### 3. Summary of the change 

 add npu_dpu_groups=6 for qwen layer_num=3.

### 4. How to test?
- [ ] Unit test: Please manually trigger the PR Validation [here](https://github.com/intel-analytics/ipex-llm-workflow/actions/workflows/llm-PR-validation.yml) by inputting the PR number (e.g., `1234`). And paste your action link here once it has been successfully finished.

### CODE REVIEW

**Review by cyita — APPROVED** [review]
LGTM

File: python/llm/src/ipex_llm/transformers/npu_models/qwen2_mp.py [hunk 1]
```diff
@@ -234,7 +234,7 @@ def __init__(
         print(f"{mode} start compiling")
         if (
             group_size != 0
-            and (mode == "prefill" or num_layers == 2)
+            and (mode == "prefill" or num_layers == 2 or num_layers == 3)
```
Comment (jason-dai) [inline python/llm/src/ipex_llm/transformers/npu_models/qwen2_mp.py:237]: We should unify the compilation configs with that in `update_names_of_IR_and_export_blob`

Comment (jason-dai) [reply python/llm/src/ipex_llm/transformers/npu_models/qwen2_mp.py:237]: We can do that in a future PR.

**Review by jason-dai — APPROVED** [review]
LGTM

File: python/llm/src/ipex_llm/transformers/npu_models/qwen2_mp.py (hunk 1 above)
Comment (rnwang04) [reply python/llm/src/ipex_llm/transformers/npu_models/qwen2_mp.py:237]: Got it, will try to figure out how to do it and support it in next PR.

**Pull request merged**